### DISCUSSION TITLE
linux-fs docs?

### DISCUSSION THREAD

**Discussion by RonaldTHerman** [Pi4J/pi4j#329] [General]
I realize that the linux-fs project is probably still under development but would it be possible to at least get the method declarations for readRegister and writeRegister? There appears to be more than one for each.

**Comment by RonaldTHerman** [comment]
found it

**Reply by FDelporte** [reply]
I'm always interested if I can improve the docs on pi4j.com. Is the info you needed missing in the documentation? How can we improve?

**Reply by RonaldTHerman** [reply]
It's not pi4j that I'm commenting on. I'd like to see the same java standard doc files for linux-fs.. I did find the source for it squirreled away on the jar_file download page jar-download.com and they seem to be formatted for doc file generation. I found what I wanted to know there.  

My current problem is that the linux-fs provider can't be found at runtime. (discussion #330) I thought that would be the easy thing to get right. :-)

**Reply by RonaldTHerman** [reply]
Actually I just found the doc files I was looking for.. not much there yet.
